common/src/main/java/org/jboss/hal/testsuite/fragment/config/messaging/MessagingPropertyWizard.java

```diff
@@ -0,0 +1,32 @@
+package org.jboss.hal.testsuite.fragment.config.messaging;
+
+import org.jboss.hal.testsuite.fragment.shared.modal.WizardWindow;
+
+/**
+ * Class representing dialog displayed when user wants to add new property in messaging configuration subsystem
+ */
+public class MessagingPropertyWizard extends WizardWindow {
```

Comment:
This looks like duplicate of `org.jboss.hal.testsuite.fragment.config.resourceadapters.ConfigPropertyWizard` shouldn't these two be unified in one?